src/providers/ecency/ecency.ts

```diff
@@ -116,20 +116,22 @@ export const deleteDraft = async (draftId: string) => {
 };
 
 /**
- * @params title
- * @params body
- * @params tags
- * @param meta
+ * @param draft
  */
-export const addDraft = async (title: string, body: string, tags: string, meta: Object) => {
+export const addDraft = async (draft: Object) => {
+  const { title, body, tags, meta, action } = draft;
   try {
     const data = { title, body, tags, meta };
     const res = await ecencyApi.post('/private-api/drafts-add', data);
-    const { drafts } = res.data;
-    if (drafts) {
-      return drafts.pop(); //return recently saved last draft in the list
-    } else {
-      throw new Error('No drafts returned in response');
+    if (action === 'add') {
```

Comment:
using exclusive property `action` may be unnecessary and confusing for later use... why not just return the full response data array, `editorContainer` can just post process to extract last draft item....